sdk/examples/noop_python/sawtooth_noop/client_cli/create_batch.py

```diff
@@ -31,7 +30,7 @@
 class NoopPayload(object):
     def __init__(self):
         self.nonce = binascii.b2a_hex(random.getrandbits(
-            8 * 8).to_bytes(8, hbyteorder='little'))
+            8 * 8).to_bytes(8, byteorder='little'))
```

Comment:
This is a different type of change than the rest of the commit; suggest it be in a different commit with an alternate explanation if it's not an accident.

Reply:
As requested, I reintroduced the hbyteorder typo.

Then I separated the s/hbyteorder/byteorder/ fix into a separate fix.